libr/core/cmd_print.c

```diff
@@ -1774,7 +1774,8 @@ static void cmd_print_format(RCore *core, const char *_input, const ut8* block,
 			if (val) {
 				r_cons_printf ("%d\n", r_print_format_struct_size (core->print, val, mode, 0));
 			} else {
-				eprintf ("Struct %s not defined\nUsage: pfs.struct_name | pfs format\n", _input);
+				R_LOG_ERROR ("Struct %s not defined", _input);
```

Comment:
combine the two messages into a single string like:
```
R_LOG_ERROR ("struct %s is not defined. Use pfs.struct|pfs format");
```

no need to waste two lines for thsi